packages/shared/src/components/auth/AuthOptions.tsx

```diff
@@ -192,6 +197,12 @@ function AuthOptions({
     onSocialRegistration(provider);
   };
 
+  const onForgotPasswordSubmit = (_email: string, _flow: string) => {
```

Comment:
```suggestion
  const onForgotPasswordSubmit = (email: string, flow: string) => {
```

I think _ should only be used for `un-used` parameters

Reply:
Made a suggestion 🙏